runtime/server/queries_downloads.go

```diff
@@ -0,0 +1,23 @@
+package server
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+func (s *Server) Download(ctx context.Context, req *runtimev1.DownloadLinkRequest) (*runtimev1.DownloadLinkResponse, error) {
```

Comment:
Let's move this func to the `downloads.go` file so we keep the download logic together in one file